src/directives/layers.js

```diff
@@ -81,9 +81,9 @@ angular.module("leaflet-directive").directive('layers', function ($log, $q, leaf
                             }
                             delete leafletLayers.baselayers[name];
 
-                            // if (newBaseLayers[name].doRefresh) {
-                            //     newBaseLayers[name].doRefresh = false;
-                            // }
+                            if (newBaseLayers[name] && newBaseLayers[name].doRefresh) {
```

Comment:
@nmccready, I was surprised about this like you, but it seems that the newBaseLayers[name] item can be undefined. The tests didn't pass if it's not present that line.

I was surprised that travis didn't catch this too. Do you know why this could have happened?

![screenshot from 2015-04-25 15-28-07](https://cloud.githubusercontent.com/assets/124169/7333066/b619b314-eb5f-11e4-96e4-103bd76d64ff.png)